tests/rest/client/test_relations.py

```diff
@@ -486,12 +487,13 @@ def test_bundled_aggregations(self):
         self.assertEquals(200, channel.code, channel.json_body)
         thread_2 = channel.json_body["event_id"]
 
-        def assert_bundle(actual):
+        def assert_bundle(event_json: JsonDict) -> None:
             """Assert the expected values of the bundled aggregations."""
+            relations_dict = event_json["unsigned"].get("m.relations")
 
             # Ensure the fields are as expected.
             self.assertCountEqual(
```

Comment:
this assertion function always gets me with the misleading name :face_with_spiral_eyes:

Reply:
Definitely a confusing name. 🤷